Makefile

```diff
@@ -22,13 +22,16 @@ update: install # update all python dependencies
 	pipenv update --dev
 
 ## ---- Test commands ---- ##
-test: # run tests and print coverage report
-	pipenv run coverage run --source=lambdas -m pytest -vv
+test: # Run tests and print a coverage report
+	pipenv run coverage run --source=lambdas -m pytest -vv -m "not integration"
 	pipenv run coverage report -m
 
 coveralls: test
 	pipenv run coverage lcov -o ./coverage/lcov.info
 
+test-integration:
+	pipenv run pytest -vv -s -m "integration"
```

Comment:
What do each of these flags do: `-vv`, `-s`, `-m`? 🤔

Reply:
* `-vv`: verbose
  * `-s`: [captures stdout output](https://docs.pytest.org/en/latest/how-to/capture-stdout-stderr.html#setting-capturing-methods-or-disabling-capturing); more chatty, but can be helpful
  * `-m "integration"`: `-m` is for markers, allowing for filtering of tests based on any marks added

These new marks are defined here: https://github.com/MITLibraries/alma-webhook-lambdas/blob/GH-ISS-147-manual-integration-tests/pyproject.toml#L11-L14.

And you can see them decorating tests here (as an example): https://github.com/MITLibraries/alma-webhook-lambdas/blob/GH-ISS-147-manual-integration-tests/tests/integration/test_webhook.py#L61

So if you ran:
```shell
pytest -m "not integration"
```

Any of those integration tests would be skipped, which is what's done here: https://github.com/MITLibraries/alma-webhook-lambdas/blob/GH-ISS-147-manual-integration-tests/Makefile#L26.

Reply:
Awesome, thank you!